### ISSUE TITLE
Verification of Email IDs

### ISSUE THREAD

**Issue by suyashXD** [reedbarger/react-slack-clone#2]
Is there a way for email verification using firebase in the register.js file?

**Comment by borlePrathmesh** [comment]
Yes firebase has authentication features providing email verification, you can find the documentation here [https://firebase.google.com/docs/auth/web/email-link-auth#:~:text=Enable%20Email%20Link%20sign%2Din%20for%20your%20Firebase%20project,-To%20sign%20in&text=In%20the%20Firebase%20console%2C%20open,in)%20sign%2Din%20method.](url)
